### PR TITLE
Native ActionSheet implementation for 3dots menu, and MyOffrz integration

### PR DESCRIPTION
**Pull request by naira-cliqz**

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -347,6 +347,7 @@ class BrowserViewController: UIViewController {
         urlBar = URLBarView()
         */
         urlBar = CliqzURLBar()
+		NotificationCenter.default.addObserver(self, selector: #selector(urlBarDidPressPageCliqzOptions), name: URLBarDidPressPageOptionsNotification, object: nil)
```
Comment (mahmoud-cliqz) [inline Client/Frontend/Browser/BrowserViewController.swift]: could you fix the indentation of this line

File: Cliqz/URLBar/CliqzURLBar.swift [hunk 2]
```diff
@@ -145,6 +155,9 @@ class CliqzURLBar: URLBarView {
         }
         
     }
+	
+	override func tabLocationViewDidTapPageOptions(_ tabLocationView: TabLocationView, from button: UIButton) {		self.delegate?.urlBarDidPressCliqzPageOptions(self, from: tabLocationView.pageOptionsButton)
```
Comment (mahmoud-cliqz) [inline Cliqz/URLBar/CliqzURLBar.swift]: could you add new line to method body

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (naira-cliqz) [reply Client/Frontend/Browser/BrowserViewController.swift]: The indentation is fine, github  messes up...

File: Cliqz/URLBar/CliqzURLBar.swift (hunk 2 above)
Comment (naira-cliqz) [reply Cliqz/URLBar/CliqzURLBar.swift]: This one is fixed

**Pull request merged**